packages/react-native/scripts/honeybadger-upload-sourcemaps.sh

```diff
@@ -0,0 +1,317 @@
+#!/bin/bash
+
+# ----------------------------------------------------------------------------
+#
+# Generate and upload sourcemaps to Honeybadger
+#
+# USAGE:
+# npx honeybadger-upload-sourcemaps [--no-hermes] [--skip-upload] --apiKey <project-api-key> --revision <build-revision>
+#
+# Hermes is enabled by default in React Native projects as of React Native 0.70.
+# This script assumes that Hermes is being used. If you are not using Hermes in 
+# your React Native project, use the "--no-hermes" flag.
```

Comment:
It would be nice if we could figure out from the script whether the project uses hermes or not 🤔 .

Reply:
@subzero10 I have not been able to figure out how to do this yet, but I can take a look further into this. I created https://github.com/honeybadger-io/honeybadger-js/issues/1183